Tools/autotest/common.py

```diff
@@ -5588,6 +5676,494 @@ def test_gripper(self):
         self.context_pop()
         self.reboot_sitl()
 
+    def test_set_position_global_int(self, test_alt=True, test_heading=False, test_yaw_rate=False, timeout=100):
```

Comment:
Please comment why we have these parameters.

Also, explain why we don't use `is_rover()` etc... I can't see why we shouldn't.